web/src/popover_menus.js

```diff
@@ -441,4 +451,37 @@ export function initialize() {
             message_actions_popover_keyboard_toggle = false;
         },
     });
+
+    // Drafts popover
+    tippy_no_propagation(".drafts-sidebar-menu-icon", {
+        placement: "right",
+        popperOptions: {
+            modifiers: [
+                {
+                    name: "flip",
+                    options: {
+                        fallbackPlacements: "bottom",
+                    },
+                },
+            ],
+        },
```

Comment:
Looking at this, maybe we should extract this set of options as a `left_sidebar_popper_options` common variable in the original starred messages PR so we don't need to repeat it with each sidebar popover.